fmriprep/viz/config.json

```diff
@@ -2,14 +2,20 @@
 "sub_reports":
 [
     {
-        "name": "Anatomical",
+        "name": "Summary",
         "elements":
         [
             {
                 "name": "anat/summary",
                 "file_pattern": "anat/.*_summary",
```

Comment:
`DerivativesDataSink` makes some naming assumptions that don't make it automatic to add a new top-level namespace. I'm not sure that it's worth it, for a single-element entry.